schemas/2014-04-01-preview/deploymentTemplate.json

```diff
@@ -121,7 +121,7 @@
                   { "$ref": "https://schema.management.azure.com/schemas/2014-04-01-preview/Microsoft.Sql.json#/resourceDefinitions/administrators" },
                   { "$ref": "https://schema.management.azure.com/schemas/2014-04-01-preview/Microsoft.Sql.json#/resourceDefinitions/disasterrecoveryconfiguration" },
                   { "$ref": "https://schema.management.azure.com/schemas/2017-09-07-privatepreview/Microsoft.Kusto.json#/resourceDefinitions/clusters" },
-                  { "$ref": "https://schema.management.azure.com/schemas/2017-09-07-privatepreview/Microsoft.Kusto.json#/resourceDefinitions/databases" },
```

Comment:
>databases [](start = 141, length = 9)

update in 2015-01-01/deploymentTemplate.json as well?

Reply:
Done